src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Border/BorderAntiAlias.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<Page
+    x:Class="Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.Border_AntiAlias"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Windows_UI_Xaml_Controls.BorderTests"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="White"
+    d:DesignHeight="300"
+	d:DesignWidth="400">
+
+	<Grid>
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition Width="*"/>
+			<ColumnDefinition Width="*"/>
+			<ColumnDefinition Width="Auto"/>
+		</Grid.ColumnDefinitions>
+		<Border x:Name="FirstBorder" Background="Blue" CornerRadius="40" Height="300" Width="50" Margin="20" />
```

Comment:
you can make them publicly accessible with [`x:FieldModifier="Public"`](https://learn.microsoft.com/en-us/dotnet/desktop/xaml-services/xfieldmodifier-directive) without having to add `Get_Xyz()` in the code-behind.

Reply:
Cool thank you!